README.md

```diff
@@ -0,0 +1,6 @@
+**Todos**
+
+1. Implement a interpolater to read the file
+2. Create a loop that loops through the file and per every second copy that line of text to a new array
+3. Once done, return a new Array of Datapoints Interface at Index.ts line 1
```

Comment:
Also focus on how you can you go to do the testing to make sure that the input and the output is correct.